developer_setup.md

```diff
@@ -193,14 +193,14 @@
 ```bash
 git clone git@github.com:JoeSmith/manageiq.git # Use "-o my_fork" if you don't want the remote to be named origin
 cd manageiq
-git remote add upstream git@github.com:ManageIQ/manageiq.git
+git remote add upstream https://github.com/ManageIQ/manageiq.git
```

Comment:
Since we have the SSH instructions above (line 161), I would prefer to leave this as SSH.  @bdunne ?